### PR TITLE
Revert "Merge branch 'master' into release/2.2"

### PR DESCRIPTION
**Pull request by natemcmaster**
This reverts commit 52e5852e4827b5b2f580564945367437a6a1416d, reversing
changes made to 1e807240ea4f089ddab19a75cbedd34c69d272b0.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ addons:
     - libunwind8
 branches:
   only:
-  - master
+  - dev
```
Comment (rynowak) [inline .travis.yml]: Should this still say master?

File: run.ps1 [hunk 2]
```diff
@@ -52,8 +52,8 @@ in the file are overridden by command line parameters.
 Example config file:
 ```json
 {
-  "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/master/tools/korebuild.schema.json",
-  "channel": "master",
+  "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
+  "channel": "dev",
```
Comment (rynowak) [inline run.ps1]: should this be release/2.2?

Comment (natemcmaster) [reply run.ps1]: This is just docs, so doesn't matter.

**Comment by natemcmaster** [comment]
IMO more work than it’s worth. Force push is blocked, so we’ve have to get an admin to temporarily unblock force push.

In general, my advice is to not open a PR to directly merge release/* branches to master. Create a snapshot of the release/* branches instead so you can stage merge-conflict resolutions without affecting the release/* branch.

**Pull request merged**